tsuru/client/apps.go

```diff
@@ -728,7 +716,9 @@ func (a *app) String(simplified bool) string {
 	}
 
 	var buf bytes.Buffer
-	tmpl := template.Must(template.New("app").Parse(format))
+	tmpl := template.Must(template.New("app").Funcs(template.FuncMap{
+		"LockString": LockString,
```

Comment:
nice catch @prbn021, by convention functions should have lowercase like `lockString`